akka-cluster/src/multi-jvm/scala/akka/cluster/SharedMediaDriverSupport.scala

```diff
@@ -48,7 +48,7 @@ object SharedMediaDriverSupport {
           })
           catch {
             case NonFatal(e) =>
-              println(e.getMessage)
+              println("Exception checking isDriverActive: " + e.getMessage)
```

Comment:
Is there a static logger we can use versus `println`? Nit on `println` in tests if avoidable.

Reply:
Not AFAIK, they need an actor system